lib/tasks/pulsearch.rake

```diff
@@ -40,6 +40,6 @@ namespace :pulsearch do
   private
 
     def url_for_file(file)
-      "https://raw.githubusercontent.com/pulibrary/pul_solr/main/solr_configs/catalog-production-v2/#{file}"
+      "https://raw.githubusercontent.com/pulibrary/pul_solr/main/solr_configs/catalog-production-v3/#{file}"
```

Comment:
I had to change the core name to get the tests passing in pul_solr.  This may cause issues for the tests